Formula/picat.rb

```diff
@@ -1,11 +1,10 @@
 class Picat < Formula
   desc "Simple, and yet powerful, logic-based multi-paradigm programming language"
   homepage "http://picat-lang.org/"
-  url "http://picat-lang.org/download/picat31_src.tar.gz"
-  version "3.1"
-  sha256 "c1ae1491d56e643693aa806c08c221d2cf0d59de1ddd8c31bcff1c917c979542"
+  url "http://picat-lang.org/download/picat311_src.tar.gz"
+  version "3.1#1"
```

Comment:
Why not
```suggestion
  version "3.1.1"
```

Reply:
it is actually listed as `3.1#1` on the download page

Reply:
![image](https://user-images.githubusercontent.com/1580956/130340583-b675829d-a2b7-4e61-a951-132b9316a015.png)